purchase.adoc

```diff
@@ -1,675 +1,885 @@
           "vatPercentage": 12,
-          "unitPrice": 1500,
-          "rowTaxableAmount": 1339,
-          "name": "Enkel espresso",
-          "productUuid": "32986b58-0753-11e8-bd3c-f5639f19b562",
-          "variantUuid": "32986db0-0753-11e8-bd3c-f5639f19b562",
+          "unitPrice": 10000,
+          "rowTaxableAmount": 8929,
+          "name": "T-shirt",
+          "description": "",
+          "variantName": "Small",
           "type": "PRODUCT",
-          "id": "0",
-          "autoGenerated": false,
-          "libraryProduct": true,
-          "category": {
-            "uuid": "5ed40c70-567c-11ea-ba9c-3338e0467dba",
-            "name": "Coffee"
-          }
+          "libraryProduct": true
+        },
+        {
+          "quantity": "1",
+          "productUuid": "c8037b10-5381-11eb-a35d-4df8c9434273",
+          "variantUuid": "f29e9da0-5381-11eb-9178-f532eaf25a4b",
+          "vatPercentage": 12,
+          "unitPrice": 10000,
+          "rowTaxableAmount": 8929,
+          "name": "T-shirt",
+          "description": "",
+          "variantName": "Medium",
+          "type": "PRODUCT",
+          "libraryProduct": true
         }
       ],
+      "discounts": [],
       "payments": [
         {
-          "uuid": "f091c978-bdb0-11e9-9cb5-2a2ae2dbcce4",
-          "amount": 1500,
+          "uuid": "eb0db2eb-0ab0-4650-bd9c-492ebbe36938",
+          "amount": 20000,
           "type": "IZETTLE_CASH",
           "attributes": {
-            "handedAmount": 10000
+            "handedAmount": 20000
           }
         }
       ],
-      "cashRegister": {
-        "uuid": "r_dBcr6tT2ypC1jlMuUUzg",
-        "displayName": "Kassa 1"
-      },
+      "refundedByPurchaseUUIDs": [
+        "1x6BTNcFQ-esnIMUSdX8mQ"
+      ],
       "receiptCopyAllowed": true,
+      "references": {
+        "checkoutUUID": "6b7c7b9d-eed9-4b92-b1fe-44d45c202675"
+      },
+      "created": "2021-01-13T21:34:10.968+0000",
+      "refunded": true,
+      "purchaseUUID1": "6a7d7a9c-efd8-4a93-b0ff-45d55d212774",
+      "refundedByPurchaseUUIDs1": [
+        "d71e814c-d705-43e7-ac9c-831449d5fc99"
+      ],
       "groupedVatAmounts": {
-        "12.0": 1500
+        "12.0": 20000
       },
-      "refund": false,
-      "refunded": false
+      "refund": false
     }
   ],
-  "firstPurchaseHash": "14233908814696HbDrnUNRji5iniGikNLiQ",
-  "lastPurchaseHash": "1423390928355zj9yI1wyTvqP46AG8NEaYg",
+  "firstPurchaseHash": "16105736620691x6BTNcFQ-esnIMUSdX8mQ",
+  "lastPurchaseHash": "1610573650968an16nO_YSpOw_0XVXSEndA",
   "linkUrls": [
-        "<https://purchase.izettle.com/purchases/v2?limit=10&descending=true&lastPurchaseHash=1423390928355zj9yI1wyTvqP46AG8NEaYg>; rel=\"next\""
+    "<https://purchase.izettle.com/purchases/v2?limit=2&descending=true&lastPurchaseHash=1610573650968an16nO_YSpOw_0XVXSEndA>; rel=\"next\""
   ]
 }
-```
+----
+====
+
+=== [[example3]]__Purchase with discounts returned.__
```

Comment:
I was expecting a refund with discounts on it. What is the example supposed to be?

Reply:
Good point, will remove confusing word.